configure.ac

```diff
@@ -720,6 +720,17 @@ if test "x$enable_mac_media_keys" != "xno"; then
     GENERAL_PLUGINS="$GENERAL_PLUGINS mac-media-keys"
 fi
 
+dnl Mac Now Playing
+dnl ============
+
+AC_ARG_ENABLE(mac_now_playing,
+ [AS_HELP_STRING([--enable-mac-now-playing], [enable Mac Now Playing])],
+ [enable_mac_now_playing=$enableval], [enable_mac_now_playing="no"])
```

Comment:
Same here, enable the plugin by default please. And allow explicitly disabling it with `--disable-mac-now-playing`.